ocs_ci/ocs/bucket_utils.py

```diff
@@ -88,6 +88,61 @@ def craft_s3_command(cmd, mcg_obj=None, api=False, signed_request_creds=None):
     return f"{base_command}{cmd}{string_wrapper}"
 
 
+def craft_sts_command(cmd, mcg_obj=None, signed_request_creds=None):
+    """
+    Crafts the AWS CLI STS command including the
+    login credentials and command to be ran
+
+    Args:
+        cmd: The AWSCLI STS command to run
+        mcg_obj: An MCG class instance
+        signed_request_creds: a dictionary containing AWS S3 creds for a signed request
+
+    Returns:
+        str: The crafted command, ready to be executed on the pod
+
+    """
+
+    no_ssl = (
+        "--no-verify-ssl"
+        if signed_request_creds and signed_request_creds.get("ssl") is False
+        else ""
+    )
+    if mcg_obj:
+        if mcg_obj.region:
+            region = f"AWS_DEFAULT_REGION={mcg_obj.region} "
+        else:
+            region = ""
+        base_command = (
+            f'sh -c "AWS_CA_BUNDLE={constants.SERVICE_CA_CRT_AWSCLI_PATH} '
+            f"AWS_ACCESS_KEY_ID={mcg_obj.access_key_id} "
+            f"AWS_SECRET_ACCESS_KEY={mcg_obj.access_key} "
+            f"{region}"
+            f"aws sts "
```

Comment:
The only difference I can see in `craft_sts_command` and `craft_s3_command` is string `sts`. Can we pass `sts` string to `craft_s3_command` wherever needed

Reply:
it wouldnt be s3 command if its sts. sts is completely different and would have sts specific parameters if we were to extend the functions